src/Umbraco.Core/Models/DeliveryApi/IApiContentRoute.cs

```diff
@@ -4,5 +4,7 @@ public interface IApiContentRoute
 {
     string Path { get; }
 
+    public string? QueryString { get; set; }
```

Comment:
This needs a default implementation, otherwise it'll be a binary breaking change.

Reply:
👍